src/main/scala/com/cloudera/sparkts/Lag.scala

```diff
@@ -92,4 +92,21 @@ private[sparkts] object Lag {
       }
     }
   }
+
+  private[sparkts] def lagMatTrimBoth(x: Vector[Double], outputMat: DenseMatrix[Double],
```

Comment:
Spread these out so there's an argument per line, as in the function definition above.